test/Smoking_test_suite/utils/downloaddriver.py

```diff
@@ -0,0 +1,34 @@
+#! coding=utf-8
+from utils.checkplantfram import checkos
+import os
+import requests
+from contextlib import closing
+import zipfile
+
+def unzip(sou,dst):
+    r = zipfile.is_zipfile(sou)
+    if r:
+        fz = zipfile.ZipFile(sou, 'r')
+        for file in fz.namelist():
+            fz.extract(file, dst)
+
+
+def download(downloadurl):
+    with closing(requests.get(downloadurl, stream=True)) as response:
+        chunk_size = 1024
+        with open('../resources/chromedriver.zip', "wb") as file:
+            for data in response.iter_content(chunk_size=chunk_size):
+                file.write(data)
+    unzip("../resources/chromedriver.zip", "../resources/")
+    os.remove("../resources/chromedriver.zip")
+
+def downloadDriver():
+    os_sys = checkos()
+    if os_sys.lower() == 'windows':
+        download("http://npm.taobao.org/mirrors/chromedriver/73.0.3683.68/chromedriver_win32.zip")

```

Comment:
can we use pip install this chromedriver?
refer to https://pypi.org/project/chromedriver-binary/